<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseAcadDlg</class>
 <widget class="QDialog" name="ChooseAcadDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoCAD settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tbVersion">
      <attribute name="title">
       <string>AutoCADs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Use AutoCAD configuration</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbAlwaysAsk">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <item>
            <property name="text">
             <string>Use selected</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ask if not runnig</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Always ask</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QListWidget" name="lwAutocads"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="pbCopy">
             <property name="text">
              <string>Copy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbRename">
             <property name="text">
              <string>Rename</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbRemove">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pbResetAll">
             <property name="text">
              <string>Reset all</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stack">
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QCheckBox" name="cbNoLogo">
               <property name="text">
                <string>Start with &quot;/nologo&quot;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbLoadAecBase">
               <property name="text">
                <string>Load AecBase.dbx</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Profile</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbProfile">
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Additional command line</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leCommandLine"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Max. wait for start, sec</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="leDelayAfterOpen">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Start check interval, ms</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="sbDelayAfterStart">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="minimum">
                <number>5</number>
               </property>
               <property name="maximum">
                <number>60</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="sbMaxWaitForStart">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="minimum">
                <number>5</number>
               </property>
               <property name="singleStep">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Delay after open drawing, ms</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="lblLOBBufferSize">
               <property name="text">
                <string>Oracle buffer size, MB</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="leStartCheckInterval">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Delay after start, sec</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="1">
              <widget class="QSpinBox" name="sbLOBBufferSize">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>512</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="pteFullCmdLine">
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tbBackups">
      <attribute name="title">
       <string>Backups</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="cbKeepAfterEdit">
         <property name="text">
          <string>Keep (do not remove) temp. files after editing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbKeepAllBackupsLocally">
         <property name="text">
          <string>Keep all backups locally</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChooseAcadDlg</receiver>
   <slot>Accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChooseAcadDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Accept()</slot>
  <slot>Reject()</slot>
 </slots>
</ui>
